nifi-commons/nifi-property-utils/src/main/java/org/apache/nifi/properties/BootstrapProperties.java

```diff
@@ -32,13 +33,22 @@
 
     public enum BootstrapPropertyKey {
         SENSITIVE_KEY("bootstrap.sensitive.key"),
-        HASHICORP_VAULT_SENSITIVE_PROPERTY_PROVIDER_CONF("bootstrap.protection.hashicorp.vault.conf");
+        HASHICORP_VAULT_SENSITIVE_PROPERTY_PROVIDER_CONF("bootstrap.protection.hashicorp.vault.conf"),
+        XML_CONTEXT_LOCATION_MAPPING("bootstrap.protection.xml.context.location.mapping.");
```

Comment:
Naming this property with `XML` seems to be more format-specific than necessary, what do you think about renaming this to `CONTEXT_LOCATION_MAPPING`?

Reply:
Sure, good call.